api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters-single/solutions-output/solutions-output.service.ts

```diff
@@ -21,6 +23,16 @@ export class SolutionsOutputService
     //
   }
 
+  interruptIfKilled = () => {
+    if (this.#cancelled) {
+      // TODO: do we need to cleanup anything?
```

Comment:
@hotzevzl I suspect we should review this and implement a proper cancel of this process... whatever it is that it does